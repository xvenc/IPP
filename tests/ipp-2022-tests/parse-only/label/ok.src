.IPPcode22
LABEL ok