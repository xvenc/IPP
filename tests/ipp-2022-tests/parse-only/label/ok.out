<?xml version="1.0" encoding="UTF-8"?>
<program language="IPPcode22">
 <instruction order="1" opcode="LABEL">
  <arg1 type="label">ok</arg1>
 </instruction>
</program>
